frontends/infinite-corridor/src/pages/HomePage.test.tsx

```diff
@@ -25,4 +47,97 @@ describe("HomePage", () => {
+
+    setMockResponse.get(urls.course.upcoming(), coursesResult)
+    setMockResponse.get(urls.video.new(), [])
+    setMockResponse.get(urls.popularContent.listing(), [])
+
+    renderTestApp()
+
+    const title = await screen.findByText("Upcoming Courses")
+    assertNotNil(title)
+
+    const upcomingCoursesDiv = title.closest("div")
+
+    assertNotNil(upcomingCoursesDiv)
+
+    const [course1, course2, course3, course4] = coursesResult.results
+
+    await checkLRC(upcomingCoursesDiv, course1)
+    await checkLRC(upcomingCoursesDiv, course2)
+    await checkLRC(upcomingCoursesDiv, course3)
+    await checkLRC(upcomingCoursesDiv, course4)
+
+    const prev = within(upcomingCoursesDiv).getByText("Previous")
+    const next = within(upcomingCoursesDiv).getByText("Next")
+    expect(prev).toBeInstanceOf(HTMLButtonElement)
+    expect(next).toBeInstanceOf(HTMLButtonElement)
+  })
+
+  it("renders a resource carusel with new videos", async () => {
+    const videosResult = lrFactory.makeVideosPaginated(4)
+
+    setMockResponse.post("search/", { hits: { hits: [], total: 0 } })
+
+    setMockResponse.get(urls.course.upcoming(), [])
+    setMockResponse.get(urls.video.new(), videosResult)
+    setMockResponse.get(urls.popularContent.listing(), [])
+
+    renderTestApp()
+
+    const title = await screen.findByText("New Videos From MIT")
+    assertNotNil(title)
+
+    const newVideosDiv = title.closest("div")
+
+    assertNotNil(newVideosDiv)
+
+    const [video1, video2, video3, video4] = videosResult.results
+
+    await checkLRC(newVideosDiv, video1)
+    await checkLRC(newVideosDiv, video2)
+    await checkLRC(newVideosDiv, video3)
+    await checkLRC(newVideosDiv, video4)
+
+    const prev = within(newVideosDiv).getByText("Previous")
+    const next = within(newVideosDiv).getByText("Next")
+    expect(prev).toBeInstanceOf(HTMLButtonElement)
+    expect(next).toBeInstanceOf(HTMLButtonElement)
+  })
+
+  it("renders a resource carusel with popular learning resources", async () => {
+    const popularResult = lrFactory.makeLearningResourcesPaginated(4)
+
+    setMockResponse.post("search/", { hits: { hits: [], total: 0 } })
+
+    setMockResponse.get(urls.course.upcoming(), [])
+    setMockResponse.get(urls.video.new(), [])
+    setMockResponse.get(urls.popularContent.listing(), popularResult)
+
+    renderTestApp()
+
+    const title = await screen.findByText("Popular Learning Resources")
+    assertNotNil(title)
+
+    const popularResourcesDiv = title.closest("div")
+
+    assertNotNil(popularResourcesDiv)
+
+    const [resource1, resource2, resource3, resource4] = popularResult.results
+
+    await checkLRC(popularResourcesDiv, resource1)
+    await checkLRC(popularResourcesDiv, resource2)
+    await checkLRC(popularResourcesDiv, resource3)
+    await checkLRC(popularResourcesDiv, resource4)
+
+    const prev = within(popularResourcesDiv).getByText("Previous")
+    const next = within(popularResourcesDiv).getByText("Next")
```

Comment:
**Request:** Use `getByRole("button", { name: "Previous" })`

`getByRole` is nice because it uses the accessibility tree:

<img width="1072" alt="Screenshot 2023-04-20 at 10 57 22 AM" src="https://user-images.githubusercontent.com/9010790/233405741-1132f2ca-a9a2-4d84-ae2f-c96667486933.png">

Generally it's nice to use `getByRole`...especially for buttons and links. Sometimes you try to use `getByRole`, and it doesn't work, and you realize that you accidentally used a div for a button (which won't be keyboard focusable). That's how we realized 
-  https://github.com/mitodl/ocw-hugo-themes/issues/1126

Paul was writing a playwright test and `getByRole` wasn't working, and so we realized those buttons weren't accessible.